sdk/keyvault/Azure.Security.KeyVault.Administration/generate.ps1

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
autorest.md supports multiple links, I'm told. Please try it so we don't have custom code to maintain. Use full commit-based URLs so that taking new ones is explicitly opt-in.

Reply:
You're correct! Last time I tried, it failed.